recipe/meta.yaml

```diff
@@ -73,9 +73,10 @@ test:
 
 about:
   home: https://github.com/bblanchon/pdfium-binaries
-  summary: pre-compiled binaries of the PDFium library
+  summary: pdfium-binaries (repack by Anaconda Inc.)
```

Comment:
```suggestion
  summary: repack of [pdfium-binaries](bblanchon::pdfium-binaries)
```

Reply:
Will this kind of link actually work in the summary?